### PR TITLE
refactor(sbb-radio-button-group): lit migration

### PR DESCRIPTION
**Pull request by dauriamarco**
Migrate `sbb-radio-button` and `sbb-radio-button-group` to Lit.

### CODE REVIEW

File: src/components/sbb-radio-button-group/sbb-radio-button-group.spec.ts [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+import { html } from 'lit/static-html.js';
+import './sbb-radio-button-group';
 
 describe('sbb-radio-button-group', () => {
   it('renders', async () => {
-    const { root } = await newSpecPage({
-      components: [SbbRadioButtonGroup],
-      html: '<sbb-radio-button-group />',
-    });
+    const root = await fixture(html`<sbb-radio-button-group />`);
```
Comment (TomMenga) [inline src/components/sbb-radio-button-group/sbb-radio-button-group.spec.ts:7]: In this case should be a problem, but better to be consistent
```suggestion
    const root = await fixture(html`<sbb-radio-button-group></sbb-radio-button-group>`);
```

File: src/components/sbb-radio-button-group/sbb-radio-button-group.tsx [hunk 2]
```diff
@@ -20,99 +6,159 @@ import {
...
   /**
    * Whether the radios can be deselected.
    */
-  @Prop() public allowEmptySelection = false;
+  @property({ attribute: 'allow-empty-selection', type: Boolean })
+  public get allowEmptySelection(): boolean {
+    return this._allowEmptySelection;
+  }
+  public set allowEmptySelection(value: boolean) {
+    // TODO: Validate logic
```
Comment (TomMenga) [inline src/components/sbb-radio-button-group/sbb-radio-button-group.tsx:41]:
```suggestion
```

File: src/components/sbb-radio-button-group/sbb-radio-button-group.tsx [hunk 3]
```diff
@@ -20,99 +6,159 @@ import {
...
   /**
    * Whether the radio group is disabled.
    */
-  @Prop() public disabled = false;
+  @property({ type: Boolean })
+  public get disabled(): boolean {
+    return this._disabled;
+  }
+  public set disabled(value: boolean) {
+    // TODO: Validate logic
```
Comment (TomMenga) [inline src/components/sbb-radio-button-group/sbb-radio-button-group.tsx:57]:
```suggestion
```

**Review by TomMenga — APPROVED** [review]
LGTM 👍 
Good job, just some nitpicks

File: src/components/sbb-radio-button-group/sbb-radio-button-group.e2e.ts [hunk 4]
```diff
@@ -1,150 +1,162 @@
...
+      expect(firstRadio).to.have.attribute('checked');
 
-      await radio.click();
+      radio.click();
+      await element.updateComplete;
+      expect(radio).to.have.attribute('checked');
 
-      expect(radio).toHaveAttribute('checked');
-      expect(firstRadio).not.toHaveAttribute('checked');
+      await element.updateComplete;
```
[the remark this answers is not in the document]
Comment (dauriamarco) [reply src/components/sbb-radio-button-group/sbb-radio-button-group.e2e.ts:43]: If i remove this, the test will fail...

[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/components/sbb-radio-button-group/sbb-radio-button-group.e2e.ts:43]: it's very weird, that there is no action but an updatecomplete has to be waited :/

[the remark this answers is not in the document]
Comment (TomMenga) [reply src/components/sbb-radio-button-group/sbb-radio-button-group.e2e.ts:43]: It is strange... but maybe we are missing something. I'll mark this as resolved anyway

**Pull request merged**